Beats/winlogbeat/ingest/parser.yml

```diff
@@ -88,11 +89,22 @@ stages:
           winlog.activity_id: "{{json.event.winlog.activity_id | lower}}"
         filter: '{{json.event.winlog.get("activity_id") != None}}'
       - set:
-          process.hash.md5: "{{parse_kv_hashes.hash.MD5}}"
-          process.hash.sha1: "{{parse_kv_hashes.hash.SHA1}}"
-          process.hash.sha256: "{{parse_kv_hashes.hash.SHA256}}"
-          process.hash.sha384: "{{parse_kv_hashes.hash.SHA384}}"
-          process.hash.sha512: "{{parse_kv_hashes.hash.SHA512}}"
-          process.hash.ssdeep: "{{parse_kv_hashes.hash.SSDEEP}}"
-          process.hash.tlsh: "{{parse_kv_hashes.hash.TLSH}}"
-        filter: "{{parse_kv_hashes.hash != None}}"
+          process.hash.md5: "{{parse_kv_hashes.hash.MD5| lower }}"
+          process.hash.sha1: "{{parse_kv_hashes.hash.SHA1 | lower }}"
+          process.hash.sha256: "{{parse_kv_hashes.hash.SHA256 | lower }}"
+          process.hash.sha384: "{{parse_kv_hashes.hash.SHA384 | lower }}"
+          process.hash.sha512: "{{parse_kv_hashes.hash.SHA512 | lower }}"
+          process.hash.ssdeep: "{{parse_kv_hashes.hash.SSDEEP | lower }}"
+          process.hash.tlsh: "{{parse_kv_hashes.hash.TLSH | lower }}"
+          process.pe.imphash: "{{parse_kv_hashes.hash.IMPHASH | lower}}"
```

Comment:
is `lower`  needed ?

Reply:
Not necessary. I put it to uniformise because the other hash fields in winlogbeat events are usually lower cased, but I do no think this is mandatory. Do you think that I should get rid of the lower filter ?